README.md

```diff
@@ -10,53 +10,109 @@ See [docs](https://thesofproject.github.io/latest/index.html)
 ### Prerequisites
 
 * Docker
+* CMake (version >= 3.10)
 
 ### Build Instructions
 
-1. Run `autogen.sh`
+1. Create directory in checked out repo for build files:
+```
+mkdir build && cd build
+```
+
+2. Run configuration for your toolchain:
+
+Baytrail:
+
+```
+cmake -DTOOLCHAIN=xtensa-byt-elf -DROOT_DIR=`pwd`/../xtensa-root/xtensa-byt-elf ..
+```
+
+Cherrytrail:
+
+```
+cmake -DTOOLCHAIN=xtensa-cht-elf -DROOT_DIR=`pwd`/../xtensa-root/xtensa-cht-elf ..
+```
+
+Haswell / Broadwell:
+
+```
+cmake -DTOOLCHAIN=xtensa-hsw-elf -DROOT_DIR=`pwd`/../xtensa-root/xtensa-hsw-elf ..
+```
+
+Apollolake:
+
+```
+cmake -DTOOLCHAIN=xtensa-apl-elf -DROOT_DIR=`pwd`/../xtensa-root/xtensa-apl-elf ..
+```
+
+Cannonlake:
+
+```
+cmake -DTOOLCHAIN=xtensa-cnl-elf -DROOT_DIR=`pwd`/../xtensa-root/xtensa-cnl-elf ..
+```
+
+3. Apply default config for your platform.
```

Comment:
you are missing a cd .. here, the make xyz-defconfig doesn't work from the build directory

Reply:
You have `mkdir build && cd build` in first step. From there you build/configure etc. It's working dir for your configuration. It's CMake convention.

Reply:
never mind, I missed a step here.

Reply:
> you are missing a cd .. here, the make xyz-defconfig doesn't work from the build directory

Yup. With this, it works for me. Can you update the docs?

Also, just curious. What was the reason to switch to cmake instead of the good old autotools? :)

Reply:
@dbaluta def_configs and menus